docs/platforms/flutter/integrations/routing-instrumentation.mdx

```diff
@@ -107,6 +100,62 @@ final GoRouter _router = GoRouter(
 );
 ```
 
+## Time To Initial Display
+
+Time to initial display (TTID) provides insight into how long it takes your Widget to launch and draw their first frame, by adding a span for navigation to a Widget. The SDK sets the span operation to `ui.load.initial-display` and the span description to the Widget's route name followed by initial display, for example, `MyWidget initial display`.
+There are two ways to set up TTID:
+
+### Approximation
+
+The time to initial display is enabled by default and provides an approximation of TTID.
+
+### Manual
+
+The manual approach allows you to set a more accurate time to initial display manually.
+In order to report TTID manually wrap the desired widget with `SentryDisplayWidget`.
+
+```dart
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+SentryDisplayWidget(
+  child: MyWidget(),
+)
+```
+
+### Combining both approaches
+
+Since the approximation is enabled by default, you can combine both approaches to report the TTID of only certain Widgets manually where you need accurate measurements and leave the rest to be reported automatically via approximation.
+
+## Time To Full Display
+
+Time to full display (TTFD) provides insight into how long it would take your Widget to launch and load all of their content, by adding a span for each navigation to a Widget. The SDK sets the span operation to `ui.load.full-display` and the span description to the Widget's route name followed by full display, for example, `MyWidget full display`.
+
+TTFD is disabled by default. In order to use TTFD follow these steps:
+
+1. Enable the option in the SDK configuration.
+
+```dart
+import 'package:flutter/widgets.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+Future<void> main() async {
+  await SentryFlutter.init(
+    (options) => options.enableTimeToFullDisplayTracing = true,
+    appRunner: () => runApp(MyApp()),
+  );
+}
+```
+
+2. Report manually
+
+```dart
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+SentryFlutter.reportFullyDisplayed();
+```
+
+If the span finishes through the API, its status is set to `SpanStatus.OK`. If the span doesn't finish after 30 seconds, it will be finished by the SDK automatically, and its status will be set to `SpanStatus.DEADLINE_EXCEEDED`, also its duration will match the same of the Time to initial display span.
```

Comment:
```suggestion
If the span finishes through the API, its status is set to `SpanStatus.OK`. 

If the span doesn't finish after 30 seconds, it will be finished by the SDK automatically, and its status will be set to `SpanStatus.DEADLINE_EXCEEDED`. In this case, its duration will match the TTID span.
```